packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -3185,34 +3064,6 @@ void main() {
     overrides: <Type, Generator>{FeatureFlags: () => TestFeatureFlags(), Logger: () => logger},
   );
 
-  testUsingContext(
-    'plugin includes native Objective-C unit tests',
```

Comment:
Optional: instead of deleting this, you could rename it "Obj-C plugin includes native Swift unit tests" and adjust the expectations accordingly.

But it also wouldn't really matter if a feature I think almost nobody but us uses regresses for a deprecated flag, so I'm fine with axing it.